src/dev/run_precommit_hook.js

```diff
@@ -1,10 +1,17 @@
 import { run } from './run';
 
-import { lintFiles, pickFilesToLint } from './eslint';
+import * as Eslint from './eslint';
+import * as Tslint from './tslint';
 import { getFilesForCommit, checkFileCasing } from './precommit_hook';
 
 run(async ({ log }) => {
   const files = await getFilesForCommit();
   await checkFileCasing(log, files);
-  await lintFiles(log, pickFilesToLint(log, files));
+
+  for (const Linter of [Eslint, Tslint]) {
+    const filesToLint = Linter.pickFilesToLint(log, files);
+    if (filesToLint.length > 0) {
+      await Linter.lintFiles(log, filesToLint);
```

Comment:
This current implementation doesn't attempt to run tslint at all if there is an error from eslint, so if you have lint failures in both .js and .ts files, you won't know that until you attempt to commit again after fixing the .js files. It should probably run both commands and show the results from each.

Reply:
Yeah, the same is true for file casing issues and eslint. If there are casing issues eslint is never run.

I'll put up a separate PR that updates the precommit hook to clearly log errors from multiple tasks and collect those errors into an appropriate exit code, but I personally think it's out of the scope of this PR.

Reply:
Works for me.